f31-py37/Dockerfile

```diff
@@ -23,7 +23,7 @@ COPY ./s2i_assemble.patch /tmp/s2i_assemble.patch
 RUN TMPFILE=`mktemp` && \
     TMPFILE_ASSEMBLE=`mktemp` && \
     pushd "${STI_SCRIPTS_PATH}" && patch -p 1 < /tmp/s2i_assemble.patch && popd && \
-    pip3 install micropipenv[toml]==0.1.1 && \
+    pip3 install micropipenv[toml]==0.1.1 thamos==0.8.1 && \
```

Comment:
wouldn't this overlap with git installation of thamos in  assemble script?

Reply:
Good point - thamos will not be updated with the one from git repo. I'll add a flag to pip :+1:

Reply:
Done in adccf93.